views/update/update-check-your-answers.html

```diff
@@ -21,25 +21,57 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application
       {% if (appData.beneficial_owners_individual and appData.beneficial_owners_individual.length > 0)
       or (appData.beneficial_owners_corporate and appData.beneficial_owners_corporate.length > 0)
       or (appData.beneficial_owners_government_or_public_authority and appData.beneficial_owners_government_or_public_authority.length > 0) %}
-        <br><h2 class="govuk-heading-l">Beneficial owner details</h2>
-          {% if (appData.beneficial_owners_individual and appData.beneficial_owners_individual.length > 0) %}
-            {% set savedIndividualBO = appData.beneficial_owners_individual | rejectattr("ch_reference", "undefined") %}
-            {% for boi in savedIndividualBO %}
-              {% include "includes/check-your-answers/beneficial-owner-individual.html" %}
-            {% endfor %}
+      <br><h2 class="govuk-heading-l">Beneficial owners you have reviewed</h2>
+        {% if (appData.beneficial_owners_individual and appData.beneficial_owners_individual.length > 0) %}
+          {% set existingIndividualBO = appData.beneficial_owners_individual | selectattr("ch_reference", "undefined") %}
+          {% for boi in existingIndividualBO %}
+            {% include "includes/check-your-answers/beneficial-owner-individual.html" %}
+          {% endfor %}
+        {% endif %}
+        {% if (appData.beneficial_owners_corporate and appData.beneficial_owners_corporate.length > 0) %}
+          {% set existingCorporateBO = appData.beneficial_owners_corporate | selectattr("ch_reference", "undefined") %}
+          {% for boc in existingCorporateBO %}
+            {% include "includes/check-your-answers/beneficial-owner-other.html" %}
+          {% endfor %}
+        {% endif %}
+        {% if (appData.beneficial_owners_government_or_public_authority and appData.beneficial_owners_government_or_public_authority.length > 0) %}
+          {% set existingGovernmentBO = appData.beneficial_owners_government_or_public_authority | selectattr("ch_reference", "undefined") %}
+          {% for bog in existingGovernmentBO %}
+            {% include "includes/check-your-answers/beneficial-owner-gov.html" %}
+          {% endfor %}
+        {% endif %}
+
+        {% set textDisplayed = false %}
+        {% if (appData.beneficial_owners_individual and appData.beneficial_owners_individual.length > 0) %}
+          {% set savedIndividualBO = appData.beneficial_owners_individual | rejectattr("ch_reference", "undefined") %}
+          {% if (savedIndividualBO.length > 0)%}
+          {% set textDisplayed = true %}
```

Comment:
do we need this textDisplayed?

Reply:
yeah it's for the Heading - "Beneficial Owners you have added", otherwise that heading stays even with no new added BOs